### PR TITLE
feat: added reduce min onnx import

### PR DESCRIPTION
**Pull request by JachymPutta**
### Checklist

- [X] Confirmed that `run-checks all` script has been executed.
- [X] Made sure the book is up to date with changes in this PR.

### Related Issues/PRs
 
Helps with #1714 

### Changes

Reduce min import based on the reduce max

### Testing

Manual

`cargo test -p burn-import --color=always -- --color=always`
`cargo test -p onnx-tests --color=always`

### CODE REVIEW

File: crates/burn-import/onnx-tests/tests/reduce_min/reduce_min.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        super(Model, self).__init__()
+
+    def forward(self, x):
+        return (
+            # ReduceMax, keepdims=0, axes=None
+            torch.min(x),
+            # ReduceMax, keepdims=1, axes=[1]
+            torch.min(x, dim=1, keepdim=True).values,
+            # ReduceMax, keepdims=1, axes=[-1]
+            torch.min(x, dim=-1, keepdim=True).values,
```
Comment (laggui) [inline crates/burn-import/onnx-tests/tests/reduce_min/reduce_min.py:21]: I'm assuming the comments are supposed to refer to `ReduceMin`?

**Review by laggui — COMMENTED** [review]
Seems pretty similar to `ReduceMax` 😄 

LGTM, just a minor comment. Will merge afterwards.

**Comment by JachymPutta** [comment]
> Seems pretty similar to `ReduceMax` 😄

It was a great source of inspiration 😄

**Review by laggui — APPROVED** [review]
LGTM! 🙂 thanks for the contribution

**Pull request merged**